packages/api/cms-api/src/page-tree/page-tree.service.ts

```diff
@@ -261,6 +266,16 @@ export class PageTreeService {
         return pageTreeReadApi.getNodeOrFail(id);
     }
 
+    async updateNodeUpdateTime(id: string): Promise<void> {
+        const pageTreeReadApi = this.createReadApi({
+            visibility: "all",
+        });
+
+        const node = await pageTreeReadApi.getNodeOrFail(id);
+
+        await this.pageTreeRepository.persistAndFlush(node.assign({ updatedAt: new Date() }));
```

Comment:
flush ends a transaction and should usually happen once per mutation.

A service method should not have flush as unexpected side effect.